packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -60,7 +62,7 @@ import { TxRollupOriginationOperation } from '../operations/tx-rollup-originatio
 import { TxRollupBatchOperation } from '../operations/tx-rollup-batch-operation';
 import { TransferTicketOperation } from '../operations/transfer-ticket-operation';
 import { IncreasePaidStorageOperation } from '../operations/increase-paid-storage-operation';
-import { BallotOperation } from '../operations';
+import { BallotOperation, DrainDelegateOperation } from '../operations';
```

Comment:
I think we should separate the imports like the others. I think I've made a mistake on the BallotOperation import path. It should be its own line for clarity.

Reply:
Will update both, was wondering do we export them in `operations/index.ts` only for end users not for code base?